features/facebookImport/package.json

```diff
@@ -21,12 +21,13 @@
     "jszip": "^3.6.0",
     "memfs": "^3.2.2",
     "node-fetch": "^3.1.1",
+    "rollup-plugin-execute": "^1.1.1",
     "rollup-plugin-serve": "^1.1.0"
   },
   "private": true,
   "dependencies": {
-    "@polypoly-eu/poly-import": "file:../../feature-utils/poly-import",
     "@polypoly-eu/poly-analysis": "file:../../feature-utils/poly-analysis",
+    "@polypoly-eu/poly-import": "file:../../feature-utils/poly-import",
```

Comment:
Nit: Seems unrelated, but I have a hard time objecting to alphabetical sorting :P

Reply:
I saw this on the diff and eliminated but there were build errors - maybe sth on rebasing ?!

Reply:
hahah that was automatic change I suppose...just saw it on the diff 😇